website/src/utils/getSections.js

```diff
@@ -1,3 +1,3 @@
-const getSections = section => [...section.sections, ...section.components];
+const getSections = section => [].concat(section.sections, section.components);
```

Comment:
The previous syntax here was causing test failures when `section.sections` was undefined. I think the spread syntax may have changed subtly to disallow this `[...undefined]`.